.storybook/preview.js

```diff
@@ -13,8 +13,47 @@ import { setLanguage as setLanguageFromPackage } from '@cmsgov/design-system';
 // Rewire analytics events to log to the console
 window.utag = { link: console.log };
 
+const customViewports = {
+  ExtraSmall: {
+    name: 'Extra Small - 320px',
+    styles: {
+      width: '320px',
+      height: '800px',
+    },
+  },
+  Small: {
+    name: 'Small - 544px',
+    styles: {
+      width: '544px',
+      height: '800px',
+    },
+  },
+  Medium: {
+    name: 'Medium - 768px',
+    styles: {
+      width: '768px',
+      height: '800px',
+    },
+  },
+  Large: {
+    name: 'Large - 1024px',
+    styles: {
+      width: '1024px',
+      height: '800px',
+    },
+  },
+  ExtraLarge: {
+    name: 'Extra Large - 1280px',
+    styles: {
+      width: '1280px',
+      height: '800px',
+    },
+  },
+};
+
```

Comment:
The keys in this object don't affect what's displayed, so they should be in `camelCase` for consistency

```suggestion
const customViewports = {
  extraSmall: {
    name: 'Extra Small - 320px',
    styles: {
      width: '320px',
      height: '800px',
    },
  },
  small: {
    name: 'Small - 544px',
    styles: {
      width: '544px',
      height: '800px',
    },
  },
  medium: {
    name: 'Medium - 768px',
    styles: {
      width: '768px',
      height: '800px',
    },
  },
  large: {
    name: 'Large - 1024px',
    styles: {
      width: '1024px',
      height: '800px',
    },
  },
  extraLarge: {
    name: 'Extra Large - 1280px',
    styles: {
      width: '1280px',
      height: '800px',
    },
  },
};

```